optimum/intel/openvino/modeling_seq2seq.py

```diff
@@ -455,16 +779,12 @@ def clear_requests(self):
             raise ValueError(
                 "`clear_requests()` is not supported with `compile_only` mode, please initialize model without this option"
             )
-        self.encoder.request = None
-        self.decoder.request = None
-        if self.decoder_with_past is not None:
-            self.decoder_with_past.request = None
+        for submodel_name in self._ov_submodel_names:
+            getattr(self, submodel_name).request = None
```

Comment:
```suggestion
        for submodel in self.ov_submodels.values():
            submodel.request = None
```
```suggestion
        for submodel in self.ov_submodels.values():
            submodel.request = None
```

Reply:
I don't think this will work out because `submodel` is an instance of `openvino.Model` but should be an instance of `OVEncoder` or `OVDecoder`.

Reply:
> I don't think this will work out because submodel is an instance of openvino.Model but should be an instance of OVEncoder or OVDecoder.

yes you're correct @nikita-savelyevv, we can't  use `self.ov_submodels` directly for this reason unfortunately @IlyasMoutawwakil

Reply:
ooh I see thanks !